### ISSUE TITLE
Elements in use report breaks when a block contains an image shortcode with invalid width or height

### ISSUE THREAD

**Issue by robbieaverill** [dnadesign/silverstripe-elemental#664]
**Elemental 4.1.x-dev**

Example content that will break the Elements in use report:

```
<p><span>CWP 2 and SilverStripe 4 provide your agency with an intuitive, extensible CMS that is full of new features that make it faster, more secure, highly customisable, and easy-to-use for your content authors and developers.</span></p><p><span>[image src="/assets/Feature-Page-Images/910d0e396a/feature_image-v2.png" id="35" width="width" height="height" class="leftAlone ss-htmleditorfield-file image" title="feature image v2"]</span></p>
```

The `width="width"` and `height="height"` attribute values causes exceptions when viewing the report:

```
[Emergency] Uncaught InvalidArgumentException: Width must be a numeric value
GET /admin/reports/show/DNADesign-Elemental-Reports-ElementsInUseReport
Line 1044 in /Users/robbieaverill/dev/cwp23/vendor/silverstripe/assets/src/ImageManipulation.php
```

We can catch this exception, but what should we do instead? Breaking the report seems wrong.

**Comment by robbieaverill** [comment]
This would also affect the banner block module, and anyone using content fields in their own blocks. Maybe we can catch this exception in the report instead

**Comment by robbieaverill** [comment]
Fixed in #665
